### ISSUE TITLE
Font-size of textarea on about:adblock is too small

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#6851]
**Describe the issue you encountered:**
Font-size of the textarea on about:adblock is too small

<img width="658" alt="screenshot 2017-01-25 17 47 01" src="https://cloud.githubusercontent.com/assets/3362943/22283569/5b4e1f9a-e326-11e6-821e-71b5ff13ab79.png">

- Brave Version (revision SHA): f682695

- Steps to reproduce:
    1. Open about:adblock
    2. Input some strings in the input area
    
- Actual result: the default font-size, 11px, is applied

- Expected result: It should be larger

- Any related issues:

## Test plan:

1. Open about:adblock
2. Input some strings in the custom filters area
3. Make sure they are readable

**Comment by luixxiul** [comment]
Unintentionally posted during writing. I updated.

**Comment by luixxiul** [comment]
I'm setting the font-size to `95%`. @bradleyrichter let me know if it should be another value, thanks.
